### PR TITLE
Created hamburger menu with navigation

### PR DESCRIPTION
**Pull request by dashakrashchenko**
- Created hamburger menu
- Added navigation
- Added integration with Redux (created reducer and added to store)

Attached two screenshots.
hamburger-menu-closed
hamburger-menu-open

### CODE REVIEW

File: .expo/packager-info.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "devToolsPort": 19002,
-  "expoServerPort": 19000,
-  "packagerPort": 19001,
-  "packagerPid": 10012,
-  "expoServerNgrokUrl": "https://6e-w5t.anonymous.theater-schedule-mobile.exp.direct",
-  "packagerNgrokUrl": "https://packager.6e-w5t.anonymous.theater-schedule-mobile.exp.direct",
-  "ngrokPid": 10696
+  "expoServerPort": null,
```
Comment (vtorkalo) [inline .expo/packager-info.json:3]: why values removed from config?

File: .expo/packager-info.json.1857967446 [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "devToolsPort": 19002,
```
Comment (vtorkalo) [inline .expo/packager-info.json.1857967446:2]: looks like this added file also not related to navigation changes

File: Screens/ScheduleScreen.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+export default class ScheduleScreen extends Component {
+    static navigationOptions = {
+        drawerIcon: <MaterialCommunityIcons name='calendar-clock' size={25} />
+    }
+    render() {
+        return (
+            <Container>
+                <DrawerMenucIcon onPressMenuIcon={() => this.props.navigation.openDrawer()} />
+                <Content contentContainerStyle={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
```
Comment (vtorkalo) [inline Screens/ScheduleScreen.js]: extract to stylesheet

File: Navigation/CustomDrawerContentComponent.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { DrawerItems } from 'react-navigation';
+import { Container, Header, Content } from 'native-base';
+import { FontAwesome, MaterialIcons } from '@expo/vector-icons'
+
+const CustomDrawerContent = (props) => (
+    <Container>
+        <Header style={{ height: 100, justifyContent: 'space-around', alignItems: 'center', backgroundColor: '#7154b8' }}>
```
Comment (vtorkalo) [inline Navigation/CustomDrawerContentComponent.js]: and use stylesheet here

**Review by vtorkalo — APPROVED** [review]

**Review by andriy-mudrak — APPROVED** [review]

**Review by taras-tymchuk — APPROVED** [review]

**Pull request merged**